app/javascript/controllers/dark_mode_controller.js

```diff
@@ -0,0 +1,44 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+  static targets = ['document'];
```

Comment:
Semicolons aren't needed in javascript any longer so let's delete those just to be consistent in the project.

Also, you only need to set a target when you want the stimulus controller to reference an element *besides* the one that controller is defined on, so in this case you don't need the target. 

You can delete the `-target` on the `<html>` and here and then just change all the references from `this.documentTarget` to `this.element`

Reply:
Excellent suggestion. This is my first time with stimulus. I'd gone back and forth on including the target, considering the same.